contextvh/src/test/java/contextvh/actions/GetRelevantAreasBuildTest.java

```diff
@@ -0,0 +1,134 @@
+package contextvh.actions;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Test;
+
+import com.vividsolutions.jts.geom.Coordinate;
+import com.vividsolutions.jts.geom.Geometry;
+
+import contextvh.ContextEnv;
+import eis.exceptions.ManagementException;
+import eis.iilang.Identifier;
+import eis.iilang.Parameter;
+import eis.iilang.ParameterList;
+import nl.tytech.util.JTSUtils;
+import tygronenv.MyEnvListener;
+
+/**
+ * Test the GetRelevantAreasBuild class.
+ * @author Rico Tubbing
+ *
+ */
+public class GetRelevantAreasBuildTest {
+
+	private static final String STAKEHOLDERS = "stakeholders";
+	private static final String MUNICIPALITY = "MUNICIPALITY";
+	private static final String PROJECT = "project";
+	private static final Identifier PROJECTNAME = new Identifier("testutilsmap");
+
+	private static final double AREA_MUNICIPALITY = (1000 - 0) * (1000 - 0);
```

Comment:
It's `1000 * 1000` because it is an area of 1000*1000. I don't know about the `- 0`'s, I didn't write that...